fs/mmap/fs_rammap.c

```diff
@@ -176,7 +175,7 @@ int rammap(FAR struct file *filep, FAR struct mm_map_entry_s *entry,
   rdbuffer = kernel ? kmm_malloc(length) : kumm_malloc(length);
   if (!rdbuffer)
     {
-      ferr("ERROR: Region allocation failed, length: %d\n", (int)length);
+      ferr("ERROR: Region allocation failed, length: %zu\n", length);
```

Comment:
%z is C99 addition AFAIK, is it ok to use here?

Reply:
yes, since printf is provided by nuttx self. Actually, we just need avoid c99 language from compiler, the new feature for library which is already supported by NuttX is fine to use.